src/plugins/manualRowMove/manualRowMove.js

```diff
@@ -651,6 +655,7 @@ class ManualRowMove extends BasePlugin {
 
     if (movePerformed === true) {
       this.persistentStateSave();
+      this.hot.render();
```

Comment:
Is it necessary to render just before we add a selection and have to render it again?

Reply:
Selection doesn't re-render cell values (they may change after the move).